specs/Terminate Spec.md

```diff
@@ -0,0 +1,72 @@
+
+**1.2. Customers**
+
+PowerToys is mainly targeted towards Windows Power Users though it is available to users who want to experience using windows in a more efficient and productive way.
+ 
+  
+**1.3. Problem Statement and Supporting Customer Insights**
+
+Windows users need an accessible mechanism to completely kill a process when it is being unresponsive and hindering work flow. The team is still required to find solutions for:
+
+- A publicly accpetable name for this PowerToy.
+- The degree to which the process will be terminated.
+
+**1.4. Existing Solutions or Expectations**
+
+The current methods a user can close a running program in Windows include:
+- Clicking on the close button in the program
+- Closing the program via task manager
+- Using the keyboard shortcut Alt + F4 to close the program
+
+**1.5. Goals/Non-Goals**
+
+- Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
+
+- Meet customers expectation with end result of project.
+
+## 2. Definition of Success
+
+**2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
+
+The PowerToys repo currently has 200+ people watching, over 4000 stars and 109 forks on github despite having an empty repo. Also, this particular PowerToy received a rating of 3.44/5 in the survey we sent out to the community asking them to rate how useful they think it will be. After the PowerToys repo is made public, we hope to see at least a 5% increase in all the Github statistics aforementioned. On completion, a follow up survey will be sent out to the community and an increase in the rating to about 3.75/5 will be a good indicator that we met our customers satisfaction.
+
+## 3. Requirements
+
+**3.1.	Functional Requirements**
+
+Tou use this PowerToy, a user will:
+
+- Click the chosen keyboard shortcut and cursor morphs to indicate that user is in 'Terminate' mode.
+- In Terminate mode, whenever a user clicks on a window, it displays a yes/no prompt for the user to confirm that they want to close said window.
+    - Note: users can configure in their options to disable the yes/no prompt and just terminate apps instantly.
+- Once they are done terminating their apps, user presses keyboard shortcut again to leave terminate mode.
+
+![Terminate](https://raw.github.com/indierawk2k2/PowerToys-1/images/PT%20Images/Terminate%20Blurred.png "Terminate")
+
+- Keyboard shortcut suggestions
+    - Alt + Z
+    - Alt + Shift + X 
+
+**3.1.1 Public Name**
+
+Internally, the team has referred to this PowerToy as Terminate App. Some other name suggestions include:
+ - Destroy App
+ - Snap and Kill
+ - Kill tap
+ - End.
```

Comment:
This list item is unnecessary and confusing, the list ends because there aren't any more bullet points. Unless you mean that this app should just be called **End.** ?